freetypefont.mod/freetypefont.bmx

```diff
@@ -166,7 +166,41 @@ Type TFreeTypeFont Extends BRL.Font.TFont
 				
 		Local ft_face:Byte Ptr
 
-		If src.Find( "::" )>0
+		If TStream(src)
+			Local stream:TStream = TStream(src)
+			Local data:Byte[1024]

```

Comment:
You can probably afford to make this bigger. Maybe 8k? It might save on some allocations.

Reply:
This part is taken from bank.mod/bank.bmx : `LoadByteArray`. If we increased size here, we could also append a new param to this function for an initial capacity..

Reply:
Increased in latest commit.

Reply:
I guess it depends on how big a typical font file is? You kind of want to be at least as big as the average size, so on average you are then only doing one allocation.

Reply:
I think 8k is even too small then - with unicode fonts you end up with 100k+.
Might be that 32 or 64k is already "good to go".

Also memory allocation won't take part a hundred times per second - as you do not reload fonts so often :)